pages/information/operational.md

```diff
@@ -0,0 +1,159 @@
+---
+layout: category
+title: Operational Carbon Emissions
+permalink: /information/lifecycle/operational
+---
+
+# Operational Carbon Emissions 
+
+Operational carbon, also known as carbon emissions from use or usage emissions, refers to the [greenhouse gas (GHG)](/glossary#greenhouse-gases) emissions associated with the actual operation and use of a device or product over its lifetime. Operational carbon excludes emissions from the manufacturing, transportation, maintenance, consumables/replacements or end-of-life disposal phases, which instead fall under [embodied carbon](/glossary#embodied-carbon). Factors influencing operational carbon impact include the device's energy efficiency, usage patterns, life span and the [carbon intensity](/glossary#carbon-intensity) of the energy grid powering it.
```

Comment:
```suggestion
Operational carbon, also known as carbon emissions from use or usage emissions, refers to the [greenhouse gas (GHG)](/glossary#greenhouse-gases) emissions associated with the actual operation and use of a device or product over its lifetime. Operational carbon excludes emissions from the manufacturing, transportation, maintenance, consumables/replacements or end-of-life disposal phases, which instead fall under [embodied emissions](embodied). Factors influencing operational carbon impact include the device's energy efficiency, usage patterns, life span and the [carbon intensity](/glossary#carbon-intensity) of the energy grid powering it.
```
Thought this should be a link to the other information section for consistency so that you can hop back and forth between them.

Reply:
Yeah I wasn't sure where to link to here. My thinking was that the user may just want to know what the term means, and then if they wanted to see more detail the glossary links to the embodied page

Reply:
Could do either I guess as long as the two pages are consistent? I started reading the Embodied Emissions page and when I got to

> 'Embodied carbon excludes emissions associated with the usage of a device or product, which instead falls under [operational emissions](operational)'

 clicked the link and ended up on this page. To my mind it then seemed logical that the embodied carbon link would take me back so I was a little surprised I ended up at the glossary.

Reply:
Good spot, I'll link both to the glossary to keep consistent with other pages